tests/unit/executor/test_environment.py

```diff
@@ -165,9 +170,11 @@ def test_generate_step_environment_no_project_name(new_dir):
         export CXXFLAGS="-isystem {new_dir}/parts/p1/install/usr/include"
         export LDFLAGS="-L{new_dir}/stage/lib -L{new_dir}/stage/usr/lib -L{new_dir}/stage/lib/aarch64-linux-gnu"
         export PKG_CONFIG_PATH="{new_dir}/stage/usr/share/pkgconfig"
-        ## Plugin Environment
+        ## Plugin environment
         export PLUGIN_ENVVAR="from_plugin"
-        ## User Environment
+        ## Application environment
+        export APP_ENVVAR="from_app"
```

Comment:
the test name does not match what is being tested

Reply:
Removed application variable setting from this test.